src/manifests/input_manifest.py

```diff
@@ -136,6 +136,8 @@ def __init__(self, data: dict) -> None:
 
             def __to_dict__(self) -> dict:
                 return {
+                    "name": self.name
+                } if self.args is None else {
```

Comment:
You don't need to do this because `to_dict` should compact the outcome. Just return what you have in `else`.

Reply:
@dblock Actually I think we might need this. Otherwise it will output `args: None` in dashboard manifest it generated.

Reply:
Make sure we call `to_dict`, which [removes these `None` values](https://github.com/opensearch-project/opensearch-build/blob/794088835319cfd015a107e77f99d3a0b31efad1/src/manifests/manifest.py#L96).